codecov.yml

```diff
@@ -0,0 +1,22 @@
+# Customizations to codecov for Trillian repo. This will be merged into
+# the team / default codecov yaml file.
+#
+# Validate changes with:
+# curl --data-binary @codecov.yml https://codecov.io/validate
+
+# Exclude code that's for testing, demos or utilities that aren't really
+# part of production releases.
+ignore:
+  - "cmd/createtree/testonly"
```

Comment:
Could we not use a pattern like "**/testonly" to match all of the testonly directories?

Reply:
According to the docs the recursive match is somewhat limited. It's possible that some variant can be made to work but there's a caveat on the docs here: https://docs.codecov.io/docs/ignoring-paths

Reply:
Isn't that caveat just that "*" doesn't match recursively but "**" does?